### ISSUE TITLE
osx pip3 install Attic - command not found

### ISSUE THREAD

**Issue by uptownhr** [jborg/attic#331]
on OSX, after installing with `pip3 install Attic`

Running `attic ....` results in command not found

**Comment by emory** [comment]
I can help! I don't know when it happened but something between XCode releases and developer releases and homebrew and who knows what else made it impossible for 'pip3 install attic' to work either. I had a similar issue and looking at the `/usr/local/bin/attic` it was running the wrong python (`python` is python 2 on my system) and even if you make it use python3 it was failing and seg faulting.

tl;dr: pull the git repository, install the dependency mentioned in the README, and `python3 setup.py install` will work and leave you with a functional `attic`.
